gradle.properties

```
@@ -47,7 +47,7 @@ stdlibAuthVersion=2.10.0
 stdlibJwtVersion=2.10.0
 stdlibOAuth2Version=2.10.0
 
-stdlibHttpVersion=2.10.0
+stdlibHttpVersion=2.10.3-20231011-163800-7c8069a
```

Update version after release
```suggestion
stdlibHttpVersion=2.10.3
```